src/Controls/src/Core/HandlerImpl/Button/Button.Windows.cs

```diff
@@ -33,5 +59,19 @@ public static void MapImageSource(IButtonHandler handler, Button button)
 
 		public static void MapText(ButtonHandler handler, Button button) =>
 			MapText((IButtonHandler)handler, button);
+
+		void OnButtonSizeChanged(object sender, SizeChangedEventArgs e)
+		{
+			if (Handler is ButtonHandler buttonHandler)
+			{
+				var platformView = buttonHandler.PlatformView;
+				var virtualView = buttonHandler.VirtualView as Button;
+
+				if (platformView is null || virtualView is null)
+					return;
+
+				platformView.UpdateContentSize(virtualView);
```

Comment:
Idea, but take it or leave it if you want:
```suggestion
			if (Handler is ButtonHandler buttonHandler && buttonHandler.PlatformView is not null &&  buttonHandler.VirtualView is Button virtualView)
			{
				buttonHandler.PlatformView.UpdateContentSize(virtualView);
```